Source/utils/soundsample.cpp

```diff
@@ -16,10 +16,48 @@
 #include "storm/storm_sdl_rw.h"
 #include "storm/storm.h"
 #include "utils/log.hpp"
+#include "utils/math.h"
 #include "utils/stubs.h"
 
 namespace devilution {
 
+namespace {
+
+constexpr float LogBase = 10.0f;
+
+// Scaling factor for attenuating volume.
+// Picked so that a volume change of -10 dB results in half perceived loudness.
+// VolumeScale = -1000 / log(0.5)

```

Comment:
We use Doxygen for documentation
```suggestion
/**
 * Scaling factor for attenuating volume.
 * Picked so that a volume change of -10 dB results in half perceived loudness.
 * VolumeScale = -1000 / log(0.5)
 */
```